src/routes/session.js

```diff
@@ -0,0 +1,42 @@
+const KoaRouter = require('koa-router');
+
+const router = new KoaRouter();
+
+router.get('session.new', '/new', async (ctx) => {
+  await ctx.render('session/new', {
+    submitPath: ctx.router.url('session.create'),
+    });
+  }
+);
+
+router.post('session.create', '/', async (ctx) => {
+  const { email, password } = ctx.request.body;
+  const user = await ctx.orm.user.findOne({ where: { email } });
+
+  /* Explain, that usually you encrypt the password,
+  so you check if the input one encripted its equal to the actual one */
```

Comment:
No entendí bien este texto, así que hago una sugerencia, pero si está incorrecta puedes omitirla:
```suggestion
  so you check if the encrypted input is equal to the actual password */
```